.travis.yml

```diff
@@ -12,7 +12,7 @@ matrix:
 install:
   # newer versions of PyYAML dropped support for Python 3.4
   - if [ $TRAVIS_PYTHON_VERSION == "3.4" ]; then pip install PyYAML==5.2; fi
-  - pip install coverage flake8 flake8-docstrings flake8-import-order pytest PyYAML
+  - pip install appdirs coverage flake8 flake8-docstrings flake8-import-order pytest PyYAML mock
```

Comment:
The dependency on `appdirs` needs to be declared in the `setup.py` as well as `stdeb.cfg` file.

Reply:
Whoops, thank you.

Reply:
Fixed in 92a3bdf8e4183469bad475e67a0a1af4d8695855.